lang/resources/org/partiql/type-domains/partiql.ion

```diff
@@ -126,13 +126,8 @@ may then be further optimized by selecting better implementations of each operat
             (date year::int month::int day::int)
             (lit_time value::time_value)
 
-            // Set operators
-            (union setq::set_quantifier operands::(* expr 2))
-            (except setq::set_quantifier operands::(* expr 2))
-            (intersect setq::set_quantifier operands::(* expr 2))
-            (outer_union setq::set_quantifier operands::(* expr 2))
-            (outer_except setq::set_quantifier operands::(* expr 2))
-            (outer_intersect setq::set_quantifier operands::(* expr 2))
```

Comment:
I think these changes are backwards-incompatible and warrants calling out in the `CHANGELOG.MD` since users using the "unrefactored" AST may need to change code on their end.

Also will need to update the PR description accordingly.